### ISSUE TITLE
Datatable is not an NgModule... Anyone has fixed this issue...

### ISSUE THREAD

**Issue by AshwinDeva** [ggmod/angular-2-data-table#20]

**Comment by mehtasagar** [comment]
Me too facing the same problem. Did you found the solution?

**Comment by AshwinDeva** [comment]
No... Do anyone have work around to compile successfully till fixing it..?

**Comment by AshwinDeva** [comment]
As a workaround,  in index.d.ts file of angular-2-data-table, add the below lines ,

import { NgModule } from 'angular/core';
@NgModule({}) export declare class DataTableModule {
}

**Comment by ericalbers** [comment]
when adding those lines, throws errors about resolving symbols statically
This is still an open bug.

**Comment by ericalbers** [comment]
Switching to another table solution, this one does not work.

**Comment by uakgul-talis** [comment]
you don't have to "add" those lines, just add the "@NgModule({})".
The export declare class DataTableModule {
} is already here, just add the @NgModule part. 
Worked for me.

**Comment by nitthi** [comment]
i have a same problem, can anyone please help on this issue or suggest any other table solution like angular-2-data-table.
